tests/screenshot/apps/components/Slider.js

```diff
@@ -2,10 +2,11 @@ import Slider, {SliderTooltip as Tooltip} from '../../../../Slider';
 
 import css from './Slider.module.less';
 
-// TODO: RTL, different min/max with visible tooltip
-
 const SliderTests = [
 	<Slider />,
+	<Slider disabled />,
+	<Slider active />,
```

Comment:
This `active` prop has no effect when being given by the user. It is handled by the hoc.
The slider is the same as the default one.

![image](https://user-images.githubusercontent.com/63335068/118933926-70736680-b952-11eb-84e2-2d5af84c1f7a.png)